README.md

```diff
@@ -1 +1,190 @@
-# react-todo-app
\ No newline at end of file
+<a id="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+    <br>
+  <h1 style="font-size: 35px; color: lightblue"><b>React-Todo-App</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+
+# 📗 Table of Contents <a id="table-of-contents"></a>
+
+- [� Table of Contents](#table-of-contents)
+- [📖 React-Todo-App ](#about-project)
+- [🛠 Built With ](#built-with)
+  - [Tech Stack ](#tech-stack)
+  - [Key Features ](#key-features)
+- [🚀 Live Demo ](#live-demo)
+- [💻 Getting Started ](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors ](#authors)
+- [🔭 Future Features ](#future-features)
+- [🤝 Contributing ](#contributing)
+- [⭐️ Show your support ](#️support)
+- [🙏 Acknowledgments ](#acknowledgements)
+- [❓ FAQ (OPTIONAL) ](#faq)
+- [📝 License ](#license)
+
+<br>
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 React-Todo-App <a id="about-project"></a>
+
+**React-Todo-App** is a website with a todo list functionalities which includes, users having the ability to add a task, edit a task and delete a task.
+
+- Add future task to complete.
+- Checkout already completed tasks.
+
+## 🛠 Built With <a id="built-with"></a>
+
+### Tech Stack <a id="tech-stack"></a>
+
+<details>
+  <summary>Technology</summary>
+  <ul>
+    <li><a href="https://en.wikipedia.org/wiki/HTML">HTML</a></li>
+    <li><a href="https://en.wikipedia.org/wiki/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">Javascript</a></li>
+    </ul>
+</details>
+
+<details>
+  <summary>Tools</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">VSCode</a></li>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://reactjs.org">React.js</a></li>
+  </ul>
+</details>
+<br/>
+<!-- Features -->
+
+### Key Features <a id="key-features"></a>
+
+- **Users have the ability to add a task, edit and delete**
+- **Users have the ability to strike through tasks when completed**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a id="live-demo"></a>
+
+- An implementation of this portfolio will be live on [Github Pages](https://skabeo.github.io/reaact-todo-app/) (to be updated)
+
```

Comment:
**Changes to the live preview section!** 

- Kindly fix the live preview link, it is currently broken. You can tell the users that the live preview will be up soon.
- The implemented project is a to-do app and not a portfolio project.